.github/workflows/sync-branches.yml

```diff
@@ -0,0 +1,44 @@
+name: Sync Main to dependabotchanges
+
+on:
+  # Schedule the sync job to run daily or customize as needed
+  schedule:
+    - cron: '0 1 * * *'  # Runs every day at 1 AM UTC
+  # Trigger the sync job on pushes to the main branch
+  push:
+    branches:
+      - main
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
```

Comment:
@Harmanpreet-Microsoft This seems little unusual to me this 'ubuntu-latest' so just want to double check what is this and why it is needed?

Reply:
@Vinay-Microsoft 
the line  runs-on: ubuntu-latest line specifies the environment in which your GitHub Actions job will run. GitHub provides a set of virtual machines (runners) that can execute workflows, and ubuntu-latest is a virtual machine that runs the latest stable release of Ubuntu. It’s the default choice in GitHub Actions because it’s lightweight, fast, and works well for a majority of use cases, especially for tasks like repository syncing, building, testing, and deploying applications.